### PR TITLE
Add flutter_web_preview template

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
CC @DanTup @devoncarew

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@
 - All templates:
   - Set min SDK version to 2.2.0.
 
+- Added `flutter-web-preview` template.
```
Comment (DanTup) [inline CHANGELOG.md]: There are currently two entries in changelog, 3.2.1 and 3.3.0-dev. Should these be merged together? (I presume there won't be a 3.2.1 now?).

File: lib/src/generators/flutter_web_preview.dart [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2015, Google Inc. Please see the AUTHORS file for details.
```
Comment (DanTup) [inline lib/src/generators/flutter_web_preview.dart]: 2015? 😁

File: templates/flutter-web-preview/pubspec.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  flutter_web: any
+  flutter_web_ui: any
+
+dev_dependencies:
+  build_runner: ^1.4.0
+  build_web_compilers: ^2.0.0
+
+dependency_overrides:
+  flutter_web:
+    git:
```
Comment (DanTup) [inline templates/flutter-web-preview/pubspec.yaml]: Will these still be git refs or is this going on pub?

Comment (kevmoo) [reply templates/flutter-web-preview/pubspec.yaml]: It'll be git for the preview. We don't want to publish.

File: CHANGELOG.md (hunk 1 above)
Comment (kevmoo) [reply CHANGELOG.md]: ack!

File: lib/src/generators/flutter_web_preview.dart (hunk 2 above)
Comment (kevmoo) [reply lib/src/generators/flutter_web_preview.dart]: ack!

File: templates/flutter-web-preview/pubspec.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+name: __projectName__
+
+environment:
+  # You must be using Flutter >=1.5.0 or Dart >=2.3.0
+  sdk: '>=2.3.0-dev.0.1 <3.0.0'
+
+dependencies:
```
Comment (devoncarew) [inline templates/flutter-web-preview/pubspec.yaml]: This needs a dep on `package:pedantic` (it's referenced from the analysis options file).

Comment (kevmoo) [reply templates/flutter-web-preview/pubspec.yaml]: run w/ it if you wouldn't mind, @devoncarew

File: lib/src/generators/flutter_web_preview.dart [hunk 5]
```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2019, Google Inc. Please see the AUTHORS file for details.
+// All rights reserved. Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+import '../common.dart';
+part 'flutter_web_preview.g.dart';
+
+/// A generator for an Angular 2 application.
```
Comment (kevmoo) [inline lib/src/generators/flutter_web_preview.dart]: Ah poo – should fix this, too!

Comment (devoncarew) [reply lib/src/generators/flutter_web_preview.dart]: done!

**Comment by kevmoo** [comment]
Thank you so much, @devoncarew !

**Comment by devoncarew** [comment]
Happy to! I think the build will go green now. Signing off, but feel free to merge if this looks good.

**Pull request merged**